commands/release.go

```diff
@@ -386,7 +394,16 @@ func downloadRelease(cmd *Command, args *Args) {
 	release, err := gh.FetchRelease(project, tagName)
 	utils.Check(err)
 
+	include := args.Flag.Value("--include")
+	includeRe, err := regexp.Compile(include)
```

Comment:
If the pattern is indeed a "glob", I had expected that it follows glob rules, not golang regexp rules. Glob patterns are defined roughly (example taken from `man bash`):

> `*`      Matches any string, including the null string.
> `?`      Matches any single character.
> `[...]`  Matches any one of the enclosed characters.

Reply:
I started to look into that but couldn't find a way to do it in go without downloading a [new library](https://godoc.org/github.com/gobwas/glob) or re-implementing that myself.

Reply:
nevermind filepath.Match does exactly what this needs

Reply:
@tomlazar You can use stdlib for this https://golang.org/pkg/path/filepath/#Match

Reply:
@mislav yep thanks, I found that and used it in the new commit